homeassistant/components/evohome/__init__.py

```diff
@@ -170,15 +175,18 @@ def __init__(self, hass, params) -> None:
             )
             return False
 
-        else:
-            self.tcs = \
-                client.locations[loc_idx]._gateways[0]._control_systems[0]  # noqa: E501; pylint: disable=protected-access
+        self.tcs = client.locations[loc_idx]._gateways[0]._control_systems[0]  # noqa: E501; pylint: disable=protected-access
 
         _LOGGER.debug("Config = %s", self.config)
+        if _LOGGER.isEnabledFor(logging.DEBUG):
+            # don't do an I/O unless required
+            _LOGGER.debug(
+                "Status = %s",
+                client.locations[loc_idx].status()[GWS][0][TCS][0])
 
         return True
 
-    async def _load_auth_tokens(self) -> Tuple[str, str, datetime]:
+    async def _load_auth_tokens(self) -> Awaitable[Tuple[str, str, datetime]]:
```

Comment:
typing is wrong

Reply:
Hmm... any hints for this jobbing, hobby-level, self-trained, python wannabe?

Or: 
```python
Awaitable[Tuple[Optional[str], Optional[str], Optional[datetime]]]
```

Reply:
Or:
```python
Coroutine[Tuple[Optional[str], Optional[str], Optional[datetime]]]
```

Reply:
OK, resolving cause I am sure it is still `Awaitable`.

Reply:
The return value is not a coroutine, so it shouldn't be an awaitable, right? This function returns a tuple.

Reply:
I don't think the typing return value should include the coroutine object. The typing docs doesn't show that.

Reply:
I got confused because it is done this way:
```python
refresh_token, access_token, access_token_expires = \
    asyncio.run_coroutine_threadsafe(
        self._load_auth_tokens(), self.hass.loop).result()
```